ci/computecpp.filter

```diff
@@ -20,7 +20,7 @@ nd_item
 opencl_interop
 platform
 reduction
-specialization_constants
+spec_const
```

Comment:
What is the reason for the renaming? Does using `specialiation_constants` name causes MSVC to issue warnings?

Reply:
Would `spec_constants` work? Minor change, but slightly more understandable.

Reply:
Applied.